packages/sdk/src/content_topic.ts

```diff
@@ -0,0 +1,109 @@
+import type { Multiaddr } from "@multiformats/multiaddr";
```

Comment:
Overall I think it is the way to go - especially considering alignment on https://github.com/waku-org/pm/issues/114#issuecomment-1934535353

With current implementation I think the main problem is ambiguity between existing `filter.subscribe` and these SDK functions. 

To resolve that I think we should make these utils as close as possible to the `node` object so that they are essentially properties - this way we make sure:
- they are accessible without any additional context such as import of it or knowledge of their existence;
- consumer can use filter directly as well (which can be the case if they decide to build on top of raw RFC standard);

The most straightforward way I see is to move it to `core` and expose from [here](https://github.com/waku-org/js-waku/blob/51ec7d9510af57981cfb14803f30467deed1904e/packages/core/src/lib/waku.ts#L48) with additionally making a tech debt task. 

nb: created a `debt` label just for that

Reply:
@weboko I added a new class to SDK that extends `WakuNode` and gives it a function to create a subscription from a content topic. I think it makes sense to keep new, higher-level functions like this separate for now, and then decide on which can be moved inside the `WakuNode` class itself.

I'm not sure if the entire `WakuNode` class should be moved to sdk package just yet.

Reply:
I don't think we need to introduce another abstraction as in most cases it will be needed to subscribe by `contentTopic` 

Considering this I would move `WakuNode` all together to `sdk` and use util functions in `content_topic.ts` as private to `WakuNode` that way we keep only one facade and have additional method on the `WakuNode` to facilitate subscription by topic.

Reply:
@weboko Moved `WakuNode` from `core` to `sdk` and added a function to create a subscription in latest commit